docs/guides/ORCHESTRATION_CHAT_COMPLETION.md

```diff
@@ -259,6 +259,51 @@ try (Stream<String> stream = client.streamChatCompletion(prompt, config)) {
 Please find [an example in our Spring Boot application](../../sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/services/OrchestrationService.java).
 It shows the usage of Spring Boot's `ResponseBodyEmitter` to stream the chat completion delta messages to the frontend in real-time.
 
+
+## Add images and multiple text inputs to a message
+
+It's possible to add images and multiple text inputs to a message.
+
+### Add images to a message
+
+An image can be added to an existing message as follows.
+
+```java
+var newMessage = message.andImage("https://url.to/image.jpg");
```

Comment:
(Minor)

Please update the doc.

I want to implicitly (or explicitly) point out this only applies to `UserMessage` without mentioning this type directly.
Also "an existing message" may be to ambiguous for the reader.

```suggestion
var oldMessage = Message.user("Describe the following image");
var newMessage = message.withImage("https://url.to/image.jpg");
```